lib/lifecycles/tag.js

```diff
@@ -4,13 +4,13 @@ const checkpoint = require('../checkpoint')
 const figures = require('figures')
 const formatCommitMessage = require('../format-commit-message')
 const runExecFile = require('../run-execFile')
-const runLifecycleScript = require('../run-lifecycle-script')
+const runLifecycle = require('../run-lifecycle')
 
 module.exports = async function (newVersion, pkgPrivate, args) {
   if (args.skip.tag) return
-  await runLifecycleScript(args, 'pretag')
+  await runLifecycle({ hookName: 'pretag', args: { ...args, pkgPrivate }, version: newVersion })
   await execTag(newVersion, pkgPrivate, args)
-  await runLifecycleScript(args, 'posttag')
+  await runLifecycle({ hookName: 'posttag', args: { ...args, pkgPrivate }, version: newVersion })
```

Comment:
With [this proposal](https://github.com/conventional-changelog/standard-version/pull/669/files#r528819190) in mind, I think `pkgPrivate` would actually belong to the payload `context`, not the `args` here since we derive it from a package file when available.